### DISCUSSION TITLE
rPI PICO W - UART1

### DISCUSSION THREAD

**Discussion by ControlBlocks** [tabemann/zeptoforth#27] [Q&A] [answered]
I don't have a toolchain installed to build the uf2 for the PICO...  I was wondering if the UART1 has been tried for the console interface on a PICO W ?

**Comment by tabemann** [comment, marked as answer]
UART1 support is implemented, but the WiFi chip has no driver yet in zeptoforth. I will get to it, but I am currently working on WiFi support for the Wio RP2040 at the moment.
